dll/win32/comdlg32/filedlg.c

```diff
@@ -2774,12 +2774,12 @@ int FILEDLG95_ValidatePathAction(LPWSTR lpstrPathAndFile, IShellFolder **ppsf,
             PathAddBackslashW(lpwstrTemp);
         }
 
-        dwAttributes = SFGAO_FOLDER;
+        dwAttributes = SFGAO_FOLDER | SFGAO_FILESYSANCESTOR;
         if(SUCCEEDED(IShellFolder_ParseDisplayName(*ppsf, hwnd, NULL, lpwstrTemp, &dwEaten, &pidl, &dwAttributes)))
         {
             /* the path component is valid, we have a pidl of the next path component */
             TRACE("parse OK attr=0x%08x pidl=%p\n", dwAttributes, pidl);
-            if(dwAttributes & SFGAO_FOLDER)
+            if((dwAttributes & (SFGAO_FOLDER | SFGAO_FILESYSANCESTOR)) == (SFGAO_FOLDER | SFGAO_FILESYSANCESTOR))
```

Comment:
Insert space after `if`

Reply:
I try not to combine functional changes with whitespace changes, so it can be actually reviewed..

Reply:
Remember about suggestion features in GitHub: https://haacked.com/archive/2019/06/03/suggested-changes/

Just like this:
```suggestion
            if ((dwAttributes & (SFGAO_FOLDER | SFGAO_FILESYSANCESTOR)) == (SFGAO_FOLDER | SFGAO_FILESYSANCESTOR))
```